### ISSUE TITLE
Opening Roslyn Compilers.sln on VS 2015 Update 2 is asking for Visual Studio Update 1

### ISSUE THREAD

**Issue by eriawan** [dotnet/roslyn#12125]
**Version Used**: 
Visual Studio 2015 Update 2
**Steps to Reproduce**:

1. Fetch master
2. Open Compilers.sln
3. A popup appears. 
Screenshot:

![image](https://cloud.githubusercontent.com/assets/8773147/16222480/75bdd6d2-37c2-11e6-904f-af259c32c186.png)

**Expected Behavior**:
Should not ask for installing missing component and should not suggest to install Update 1
**Actual Behavior**:
Asking for installing missing component. Is this default behavior? If this is the intended/default behavior, is there any Wiki Github documentation for this?

cc @davkean , @richlander

**Comment by AlexanderTaeschner** [comment]
This could be a duplicate of issue #11634. Do you have the VS SDK installed for Update 2?

**Comment by eriawan** [comment]
@AlexanderTaeschner 
yes, I have VS 2015 SDK Update 2 installed. The screen above still shows.

**Comment by davkean** [comment]
@eriawan The warning is misleading and is a bug. 

Can you follow the instructions [here](https://github.com/dotnet/roslyn/blob/master/docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md#getting-the-code), in particular make sure you've run restore.cmd and built from the command-line once before opening Visual Studio.

**Comment by eriawan** [comment]
@davkean 
after reading the article you've mentioned, looks like I have to rebuild the `Roslyn.sln` manually before opening Visual Studio. 
Is this also true for `Compilers.sln`?

I'll try this now.

**Comment by eriawan** [comment]
@davkean I have tried to rebuild `Roslyn.sln` manually, and at the end of the running build, I got this error:

`\VSSDK\Microsoft.VsSDK.targets(913,5): error : CreatePkgDef : error : ProvideRoslynBindingRedirectionAttribute: Could not load specified assembly: 'System.Reflection.Metadata' reason: Could not load file or assembly 'System.Reflection.Metadata, Version=1.4.1.0, Culture=neutral, PublicKeyToken
=b03f5f7f11d50a3a' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A) [D:\GITHUB_REPO\roslyn\src\VisualStudio\Setup\VisualStudioSetup.csproj]`

Is there anything wrong or do I miss something?
